ramls/lost-item-fee-policy.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Lost Item Fee Policies
-version: v1.0
+version: v1.1
```

Comment:
Hi @OleksandrVidinieiev. is it fine to synchronize this version with the version from module descriptor (i.e. `16.3`)?

Reply:
Yes. We were going to do that eventually anyway.

Reply:
Okay, so instead of version 1.1, this should be changed to 16.3?

Reply:
Thanks! @K-Felk, we can set 16.3 here.